tests/test_users/test_routers.py

```diff
@@ -4,74 +4,76 @@
 from fastapi import status
 from fastapi.testclient import TestClient
 
-from tests.test_users.conftest import UserOutputSchema
-from src.core.factories import UserFactory
-
+from src.apps.user.schemas import UserLoginInputSchema, UserOutputSchema
+from tests.test_users.conftest import DB_USER_SCHEMA
+from src.core.factories import UserRegisterSchemaFactory
+from src.core.utils import datetime_encoder
 
 def test_create_user(
     sync_client: TestClient,
 ):
-    register_data = UserFactory.build(password="mtdqwc241", password_repeat="mtdqwc241")
-    response = sync_client.post("users/register", json=json.loads(register_data.json()))
+    register_data = UserRegisterSchemaFactory.build(password="mtdqwc241", password_repeat="mtdqwc241")
+    register_data.birth_date = datetime_encoder(register_data.birth_date)

```

Comment:
Ten encoder jest zle uzyty, bo jak bede chcial zrobic json.dumps(cos) i to cos bedzie mialo date to dalej bedzie error, a wolanie tego encodera z reki w kazdym miejscu gdzie uzywam jsona jest niepotrzebne. Powinno sie dac ustawic defaultowy encoder dla calego modulu `json`